easybuild/easyconfigs/m/MBX/MBX-1.1.0-foss-2023a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'MBX'
+version = '1.1.0'
+
+homepage = 'https://github.com/paesanilab/MBX'
+description = "MBX is an energy and force calculator for data-driven many-body simulations"
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = ['https://github.com/paesanilab/MBX/archive/']
+sources = [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+checksums = ['0f55f4950226defb46fd0814ad97f906ce4ffd6403af6817bd98cb3c68996692']
+
+builddependencies = [('Autotools', '20220317')]
+dependencies = [
+    ('Python', '3.11.3'),
+    ('SciPy-bundle', '2023.07'),
+    ('GSL', '2.7'),
+]
+
+buildininstalldir = True
+
+preconfigopts = 'echo HEEERE && echo %(installdir)s && export MBX_HOME=%(installdir)s && autoreconf -fi && '
+
+configopts = '--enable-shared --enable-verbose CXX="$CXX"'
+
+prebuildopts = 'export MBX_HOME=%(installdir)s && '
+
+preinstallopts = 'export MBX_HOME=%(installdir)s && '
```

Comment:
I'm not sure that setting `MBX_HOME` during the installation is needed at all, if not please remove

In any case, if `buildininstalldir` is not used, `MBX_HOME` should not be set to `installdir` during configure/build...